### ISSUE TITLE
What consitutes a training set? 

### ISSUE THREAD

**Issue by lordkitsuna** [AliaksandrSiarohin/first-order-model#212]
Sorry for the simple question, but ive looked over the issues and github and i am failing to understand what makes a training set a training set. If i want to make my own training set of data as i understand it i need a large amount of video, cropped to face no camera moment. There is both training and test folders tho? do i need a finished output to train against for test? or is that something else. Basically if i want to create a whole new datasets to train for what are the requirements of that data for it to be useful and how do i use it properly.

**Comment by AliaksandrSiarohin** [comment]
Yes, you need large amount of videos, like you described. 
Test set is needed if you want to evaluate how your algorithm performs on unseen data. So you can randomly split your large set of videos. Standard proportions is 90% train set and 10% test set. This is a standard machine learning procedure. If you don't want to evaluate your algorithm, or use a visual evaluation on some samples test set is not needed.
